conf/messages.en

```diff
@@ -113,8 +113,9 @@ concurrent.remove.error.multipleOfficeOfTransit.link.text = Go back to you have
 localReferenceNumber.title = What is the local reference number (LRN)?
 localReferenceNumber.heading = What is the local reference number (LRN)?
 localReferenceNumber.error.required = Enter the local reference number
-localReferenceNumber.error.length = Local reference number must be 22 characters or less
-localReferenceNumber.error.invalidCharacters = Local reference number must only include letters a to z without accents, numbers 0 to 9, hyphens and underscores
+localReferenceNumber.error.length = The local reference number must be 22 characters or less
+localReferenceNumber.error.invalidCharacters = The local reference number must only include letters a to z without accents, numbers 0 to 9, ampersands (&), apostrophes, at signs (@), forward slashes, full stops, hyphens, question marks and spaces
```

Comment:
@kenneth-rayner where did this error message come from? As far as I can see the LRN regex is `^([a-zA-Z0-9-_]{1,22})$`. No mention of any special characters other than hyphens and underscores